content/en/sensitive_data_scanner/setup/cloud_storage.md

```diff
@@ -11,12 +11,146 @@ further_reading:
+1. If you want to send AWS logs to Datadog, leave **Yes** selected.
+1. Optionally, select **Yes** if you want to use Datadog Cloud Security Management.
+1. **Enable Sensitive Data Scanner** is automatically selected by default. This tells CloudFormation to add the AWS Managed SecurityAudit policy to your Datadog AWS Integration role and enable Agentless Scanning to start scanning your cloud data stores.
+1. Click **Launch CloudFormation Template**.
+
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/data-security
+
+{{% /tab %}}
+{{% tab "Existing AWS account" %}}
+
+1. Navigate to [Sensitive Data Scanner][1] settings page.
+1. In the AWS section within **Storage**:
+    - If you have Agentless scanning already enabled in an account:
+      1. Click the pencil icon for the account.
+      1. Toggle the **Enable Sensitive Data Scanning** to add the scanner to the account.
+      1. Click **Save**.
+    - If you don't have Agentless scanning enabled in an account:
+      1. Click on the plus icon for the account you want to enable sensitive data scanning.
+      1. Select that you want to add the scanner using CloudFormation.
+      1. Select the AWS region in the dropdown menu,
+      1. Select an API key that is already configured for Remote Configuration. If the API key you select does not have Remote Configuration enabled, Remote Configuration is automatically enabled for that key upon selection.
+      1. Toggle the **Enable Sensitive Data Scanning** to add the scanner to the account.
+      1. Click **Launch CloudFormation Template**.
+
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/data-security
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Manually deploy scanners using Terraform {#manually-deploy-scanners-using-terraform}
+
+You can deploy Agentless scanners using the Terraform Module Datadog Agentless Scanner Module. See [Datadog Agentless Scanner Module][7] for more information. Datadog recommends that you choose one of these two setup options if you manually deploy scanners.
+
+- Create an AWS account dedicated to Agentless scanners. Deploy a scanner for every region that has cloud resources you want to scan.
+
+{{< img src="/security/agentless_scanning/agentless_advanced_2.png" alt="Diagram of Agentless scanning showing the Agentless scanner is deployed in a central Cloud account" width="90%" >}}
+
+- Deploy a scanner for every region that has cloud resources that you want to scan.
+
+{{< img src="/security/agentless_scanning/agentless_quickstart_2.png" alt="Diagram of Agentless scanning showing the Agentless scanner is deployed in each Cloud account" width="90%" >}}
+
+## Add scanning groups
+
+In the [Cloud Storage][6] settings page, the **Scanning Groups** section is read-only. All SDS library rules are applied within the scanning group.
```

Comment:
This doesn't seem to match the heading in that it doesn't instruct users how to add scanning groups. It sounds like it's not possible, if the section is read-only? I wonder if the heading above it is necessary, or if this can just stand alone as its own little detail.

Reply:
That's perfect! The verb was the part that was tripping me up, but without it, it makes sense 🙂 The link is a great idea too!